README.md

```diff
@@ -1,14 +1,615 @@
 # Ably Chat SDK for Android
 
-This is the repository for the Android version of the Ably Chat SDK. We aim to build the same functionality that’s available in the [JavaScript SDK](https://github.com/ably/ably-chat-js).
+<p style="text-align: left">
+    <img src="https://img.shields.io/badge/development_status-Private_Beta-ab7df8" alt="Development status"   />
+    <img src="https://badgen.net/github/license/3scale/saas-operator" alt="License" />
+</p>
+
+Ably Chat is a set of purpose-built APIs for a host of chat features enabling you to create 1:1, 1:Many, Many:1 and Many:Many chat rooms for
+any scale. It is designed to meet a wide range of chat use cases, such as livestreams, in-game communication, customer support, or social
+interactions in SaaS products. Built on [Ably's](https://ably.com/) core service, it abstracts complex details to enable efficient chat
+architectures.
 
 > [!IMPORTANT]
-> This SDK is currently in the early stages of development and is not ready to be used.
+> This SDK is currently under development. If you are interested in being an early adopter and providing feedback then you
+> can [sign up to the private beta](https://forms.gle/vB2kXhCXrTQpzHLu5) and are welcome
+> to [provide us with feedback](https://forms.gle/mBw9M53NYuCBLFpMA). Coming soon: chat moderation, editing and deleting messages.
+
+Get started using the [📚 documentation](https://ably.com/docs/products/chat).
+
+![Ably Chat Header](/images/ably-chat-github-header.png)
 
 ## Supported Platforms
 
-- Android 24 and above
+This SDK works on Android 7.0+ (API level 24+) and Java 8+.
```

Comment:
While we support `Android 7.0+ (API level 24+) and Java 8+`. 
We don't have working CI environment ( to back our claim) for java 8 or tests running on emulator API level 24+
Maybe we should add support/ create separate issue for the same.

Reply:
I think we can address this later